src/components/card/Card.tsx

```diff
@@ -75,20 +29,38 @@ export const Card = ({
   label,
   spacing = theme.space24,
 }: CardProps) => (
-  <StyledCard $spacing={spacing} className={className || ''}>
+  <div
+    className={`${styles.card} ${className || ''}`}
```

Comment:
I think it's best to use the clsx library (i think that is what we reached consensus on using) for doing anything beyond a simple className prop. Just makes it cleaner and more consistent, and funnels className logic through one place so it is easy to refactor.